### PR TITLE
docs: update file structure document

### PR DESCRIPTION
**Pull request by iRealNirmal**
Since angular 11 extra package.json file isn't generated and with version 12 strict mode is opt out bases.
Updated document to reflect the state

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: [https://github.com/angular/angular/issues/40841](https://github.com/angular/angular/issues/40841)

## What is the new behavior?
Updated document

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by iRealNirmal** [comment]
@alan-agius4 / @petebacondarwin feel free to provide any other input or suggestion in this PR

File: aio/content/guide/file-structure.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Files at the top level of `src/` support testing and running your application. S
 
 <div class="alert is-helpful">
 
-If you create an application using Angular's strict mode, you will also have an additional `package.json` file in the `src/app` directory. For more information, see [Strict mode](/guide/strict-mode).
+Creating Angular project using strict mode(flag) is opt-out bases. For more information, see [Strict mode](/guide/strict-mode).
```
Comment (petebacondarwin) [inline aio/content/guide/file-structure.md:82]: I will let @alan-agius4 chime in but I think this needs to be something more like:

```suggestion
New Angular projects use strict mode by default. If this is not desired you can opt-out when creating the project. For more information, see [Strict mode](/guide/strict-mode).
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

File: aio/content/guide/file-structure.md (hunk 1 above)
Comment (iRealNirmal) [reply aio/content/guide/file-structure.md:82]: or something

Creating new Angular projects use strict mode by default. One can opt out of using strict flag while creating project. For more information, see [Strict mode](/guide/strict-mode).

Comment (iRealNirmal) [reply aio/content/guide/file-structure.md:82]: I will also like to check content with perspective of @alan-agius4

Comment (alan-agius4) [reply aio/content/guide/file-structure.md:82]: @iRealNirmal, let's go with @petebacondarwin suggestion. Also, it appears to be outdated, although this is out-of-scope from this PR.

**Review by alan-agius4 — CHANGES_REQUESTED** [review]
Can you also please add  `Closes #40841` as commit message footer?

Thanks,

**Comment by iRealNirmal** [comment]
> Can you also please add `Closes #40841` as commit message footer?
> 
> Thanks,

@alan-agius4 will add footer, but can you confirm final edit from your side ? Or I should go with suggestion provided by @petebacondarwin

**Comment by alan-agius4** [comment]
@iRealNirmal https://github.com/angular/angular/pull/41923#discussion_r625033271

File: aio/content/guide/file-structure.md (hunk 1 above)
Comment (iRealNirmal) [reply aio/content/guide/file-structure.md]: @petebacondarwin @alan-agius4 done changes and added commit footer as well.

**Review by alan-agius4 — APPROVED** [review]
LGTM, thanks for your contribution.

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Pull request closed without merging**